src/app/api/routes/installations.py

```diff
@@ -53,13 +54,17 @@ async def delete_installation(installation_id: int = Path(..., gt=0)):
     return await crud.delete_entry(installations, installation_id)
 
 
-@router.post("/site-devices/{site_id}", response_model=List[int],
-             summary="Get all devices related to a specific site")
-async def get_all_at_given_ts_and_site(payload: Timestamp, site_id: int = Path(..., gt=0)):
+@router.get("/site-devices/{site_id}", response_model=List[int],
+            summary="Get all devices related to a specific site")
+async def get_active_devices_on_site(site_id: int = Path(..., gt=0)):
```

Comment:
We don't want to query for other timestamp? That could be useful for historical exploration though.

Reply:
I first tried it, and the serialization of the timestamp is tricky (to serialize and both to get through the pydantic validator). So since the route was created for platform needs, I figured we should first get something to work that fits their needs. And them, perhaps extend the feature later if necessary :man_shrugging:

Reply:
Ok for me. When the need arise, it should not be too complicated to implement.